frontend/src/components/molecules/PreviewTemplateDetails/PreviewTemplateDetailsSms.tsx

```diff
@@ -0,0 +1,34 @@
+import {
+  type SMSTemplate,
+  templateTypeDisplayMappings,
+} from 'nhs-notify-web-template-management-utils';
+import {
+  ContentPreview,
+  DetailSection,
+  PreviewTemplateDetailsContainer,
+  StandardDetailRows,
+} from './common';
+
+export function PreviewTemplateDetailsSms({
```

Comment:
```suggestion
export default function PreviewTemplateDetailsSms({
```
any reason all of these cant be default exports?

Reply:
All our other react components use the non-default style, however, I noticed you're switching in the eslint PR so I'm happy to match that in anticipation

Reply:
I have no string feelings on this at all - I just wish es6 javascript didn't have two different types of imports to choose from